server.go

```diff
@@ -12,12 +12,24 @@ import (
 	"github.com/blakewilliams/viewproxy/pkg/multiplexer"
 )
 
+type logger interface {
+	Fatal(v ...interface{})
```

Comment:
is `interface{}` here basically like `any` in typescript?

Reply:
Yep, exactly.

It's an interface of nothing, so everything matches. 🎉